### PR TITLE
open flexure interview

### PR DESCRIPTION
**Pull request by jcolomb**

### CODE REVIEW

File: content/blog/2023-01-31-interview-open-flexure-microscope/index.en.md [hunk 1]
```diff
@@ -0,0 +1,791 @@
...
+
+{{< expand "Show answer">}}
+It's very hard to know how many people have built it. You only know the people
+who've reported back and told you they've built it. So we did a little
+sort of survey asking on Twitter and on our forum. Have you built it? And where
+have you built it or used it? So far, it's been used on every continent,
+including Antarctica. We've got a photo of somebody using it on the ice
+in Antarctica. It's been built in about 40 countries. We also have a user forum with over
+250 people. We don't know if every single one of them has built or used
+the microscope, some people will have joined but turn off "ask questions". But the community we know about may just be the tip of the iceberg, we assume there are thousands of microscopes out there,
```
Comment (jcolomb) [inline content/blog/2023-01-31-interview-open-flexure-microscope/index.en.md:232]:
```suggestion
the microscope, some people will join only to ask questions. But the community we know about may just be the tip of the iceberg, we assume there are thousands of microscopes out there,
```

**Review by jcolomb — COMMENTED** [review]
need Julian to check

File: content/blog/2023-01-31-interview-open-flexure-microscope/index.en.md (hunk 1 above)
Comment (julianstirling) [reply content/blog/2023-01-31-interview-open-flexure-microscope/index.en.md:232]: I think "will have joined only to ask questions" is most likley what I said/meant to say

**Comment by julianstirling** [comment]
There is the one answer that is up above. Once that is fixed I am very happy for the blog post to be published.

My ORCID is 0000-0002-8270-9237

**Pull request merged**